src/Tools/MicroBuild/Build.proj

```diff
@@ -25,6 +25,8 @@
 
     <MSBuild Projects="$(ProjectDir)Roslyn.sln" BuildInParallel="true" Properties="DeployExtension=false" />
 
+    <Exec Command="$(NuGetPackageRoot)\GitLink\$(GitLinkVersion)\lib\net45\GitLink.exe $(ProjectDir) -u https://github.com/dotnet/roslyn -f Roslyn.sln -c $(Configuration)" />
```

Comment:
@GeertvanHorrik can you clarify why you think PRs are going to be difficult here?  Reading that bug it seems to be about finding the target url.  This is specified explicitly here hence detection shouldn't happen.  

The reason we want this enabled for PRs is crash dump investigation.  Our tests are configured such that in certain bad cases it will archive out our EXE / PDB combo for future investigation. Much easier to farm that investigation out to the relevant developer (not necessarily guy who make the PR) if we have source server support in the PDB.